OpenSim/Simulation/Model/Force.cpp

```diff
@@ -56,7 +56,29 @@ void Force::setNull()
 // Define properties.
 void Force::constructProperties()
 {
-    constructProperty_isDisabled(false);
+    constructProperty_appliesForce(true);
+}
+
+void
+Force::updateFromXMLNode(SimTK::Xml::Element& node, int versionNumber) {
+    if(versionNumber < XMLDocument::getLatestVersion()) {
+        if(versionNumber < 30508) {
+            // Rename property 'isDisabled' to 'appliesForce' and
+            // negate the contained value.
+            std::string oldName{"isDisabled"};
+            std::string newName{"appliesForce"};
+            if(node.hasElement(oldName)) {
+                auto elem = node.getRequiredElement(oldName);
+                elem.setElementTag(newName);
+                if(elem.getValue().find("true") != std::string::npos)
+                    elem.setValue("false");
+                else
+                    elem.setValue("true");
+            }
+        }
+    }
+
+    ModelComponent::updateFromXMLNode(node, versionNumber);
```

Comment:
Can you use`Super::` instead of `ModelComponent::`? I once had to deal with a bug where the base class type name was hard-coded in `updateFromXMLNode()` and it was very confusing to figure out.